backend/tests/integration/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
```

Comment:
we need for pytests, pytest if there is no ``__init__.py`` import the tests as top-level modules and since we're not using unique names for the tests (we organize them per module, hence we have ``search/test_views.py`` and ``records/test_views.py`` for example) pytest compains. More info: https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code